services/common/src/components/comments/MinistryComment.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from "react";
+import { timeAgo } from "@mds/core-web/common/utils/helpers";
```

Comment:
This should probably be importing from @mds/common. Interesting that it works though!